src/tracing/mod.rs

```diff
@@ -589,6 +590,37 @@ where
         None
     }
 
+    fn eofcreate(
+        &mut self,
+        context: &mut EvmContext<DB>,
+        inputs: &mut EOFCreateInputs,
+    ) -> Option<CreateOutcome> {
+        let _ = context.load_account(inputs.caller);
+        let nonce = context.journaled_state.account(inputs.caller).info.nonce;
```

Comment:
looks like this is only required for the else case, can we do this with if let else instead?